bb_server/src/world/init.rs

```diff
@@ -18,164 +20,26 @@ impl World {
-      self.commands().add(c, handle_gamemode);
-    }
+    // we should move this Code to the Proxy
 
-    let add_specific_game_mode = |name: &'static str, gm: GameMode| {
-      let mut c = Command::new(name);
-      c.add_arg_opt("target", Parser::Entity { single: false, only_players: true });
-      self.commands().add(c, move |wm, runner, args| {
-        if let Some(arg) = args.get(1) {
-          for world in wm.worlds().iter() {
-            for target in arg.entity().iter(&world.entities(), runner) {
-              if let Some(p) = target.as_player() {
-                p.set_game_mode(gm)
-              }
-            }
-          }
-        } else if let Some(player) = runner {
-          player.set_game_mode(gm);
-        } else {
-          // TODO: Send error saying they need to specify a target
-        }
-      });
-    };
-
-    add_specific_game_mode("gms", GameMode::Survival);
-    add_specific_game_mode("gmc", GameMode::Creative);
-    add_specific_game_mode("gma", GameMode::Adventure);
-    add_specific_game_mode("gmsp", GameMode::Spectator);
-
-    let c = Command::new("fly");
-    self.commands().add(c, |_, player, _| {
-      if let Some(p) = player {
-        p.set_flying_allowed(!p.flying_allowed());
-      }
-    });
-    let mut c = Command::new("flyspeed");
-    c.add_arg("multiplier", Parser::Float { min: None, max: None });
-    self.commands().add(c, |_, player, args| {
-      // args[0] is `flyspeed`
-      let v = args[1].float();
-      if let Some(p) = player {
-        p.set_flyspeed(v);
-      }
-    });
-
-    let mut c = Command::new("summon");
-    c.add_arg("entity", Parser::EntitySummon);
-    self.commands().add(c, |_, player, args| {
-      // args[0] is `summon`
-      let ty = args[1].entity_summon();
-      if let Some(p) = player {
-        let eid = p.world().summon(ty, p.pos());
-        info!("eid of mob: {}", eid);
-        p.send_message(Chat::new(format!("summoned {ty:?}")));
-      }
-    });
-
-    let mut c = Command::new("join");
-    c.add_arg("addr", Parser::String(StringType::Word));
-    self.commands().add(c, |_, player, args| {
-      if let Some(p) = player {
-        let addr = match args[1].str().parse::<std::net::SocketAddr>() {
-          Ok(a) => a,
-          Err(e) => {
-            p.send_message(Chat::new(e.to_string()));
-            return;
-          }
-        };
-        p.switch_to(SwitchMode::Loading, vec![addr]);
-      }
-    });
+    // let mut c = Command::new("join");
+    // c.add_arg("addr", Parser::String(StringType::Word));
+    // self.commands().add(c, |_, player, args| {
+    //   if let Some(p) = player {
+    //     let addr = match args[1].str().parse::<std::net::SocketAddr>() {
+    //       Ok(a) => a,
+    //       Err(e) => {
+    //         p.send_message(Chat::new(e.to_string()));
+    //         return;
+    //       }
+    //     };
+    //     p.switch_to(SwitchMode::Loading, vec![addr]);
+    //   }
+    // });
```

Comment:
can you leave this command? I use it for debugging.

Reply:
I mean for sure i can, But like i commented above, It would be better when we move this to the Proxy

Reply:
commands aren't defined in the proxy. this needs to stay as-is in the server